content/W2015/W15-blog-hollyxu.md

```diff
@@ -0,0 +1,51 @@
+
+There's a recurring theme in these interviews - taking risks. Overcoming self-esteem is a huge part of being a woman in computing. This week I sat down with Holly Xu, a fourth year student to talk about her transition into CS from accounting, how robotics club boosted her confidence, and the importance of independence.
+
+![Photo of Holly Xu](http://i725.photobucket.com/albums/ww252/itsjulianguyen/holly_xu_zps74dlwd86.png "Holly Xu")
+
+Holly had a difficult decision to make when it came to applying to university. She had always been interested in computers, but she faced familial pressure to apply for accounting, an industry with a traditionally secure job market.
+
+In high school, she was a website and t-shirt designer for the robotics club. She had always declined offers from her teammates to learn how to solder and cut metal. Despite her experience in web development, Holly felt like she was not cut out for computer science because of her inexperience with writing algorithms. She believed her interests in web development was more of an artistic hobby.
+
+Despite receiving multiple offers to engineering programs, Holly settled on the Mathematics/Chartered Professional Accountancy program at the University of Waterloo. Between her first and second year, she realized she had no passion whatsoever for accounting. She envied her engineering friends who worked on exciting software projects.
+
+"Dude why am I not like you?!" she recalls her emotions.
+
+Holly's interest in computer science grew during a first year summer job in IT. Although she anticipated disapproval from her parents, she switched programs in a rebellious act of confidence. Since she was already in the Math faculty, the switch was a straightforward process.
+
+"I realized any form that I sign at the university doesn't need parental consent. All you need is determination - visit the CS advisors," Holly reflects.
+
+As a CS student, Holly faced low self-esteem. In high school, she was accustomed to excelling with little effort. School was now a "big kick in the face" because it felt like she was competing with all the the "smart people" who had impressive grades and résumés. She didn't believe she was "smart enough" - relying heavily on her family and friends for reassurance.
+
+"Everyone here is smart - don't let that discourage you. Learn to think more independently," Holly advises.
+
+At school, Holly had no mentors she could turn to for advice on self-esteem. Holly had peers she admired but never reached out to, which she regrets.
+
+The mentorships she had during co-op helped her to develop more confidence. Holly's mentors have supported her growth as a software engineer without patronizing her for being a woman. One of her first mentors, Leith Abdulla, encouraged her to ask more questions that were intimidating to ask.
+
+"He's a great communicator," she describes, "Whenever I had problems, he would try to approach it from my perspective. It was really helpful."
```

Comment:
maybe describes -> recalls. There should be a period, not a comma, between the verb and the start of the new quote